src/main/resources/db/migration/V2024_09_20_1100__CIV-15222_bundle_restitch_task_list_scenario.sql

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
If these need running separately from the rest in the folder (for a separate release) then I think they should be in a new folder now.

Reply:
HI Sean. Thanks for your review. As far as I know and I could be wrong as well, I could not find any config that tell the flyway DB to look for scripts in a specific folder so I am assuming it takes it from the default folder ie classpath:db/migration and the migration is enabled/disabled using REFERENCE_DATABASE_MIGRATION. At the moment its enabled in preview but not in demo/prod. its enabled once when a release happens and disabled again.

Reply:
@dharmendrak Could you please confirm if this is right ?